test/std-format-test.cc

```diff
@@ -111,7 +111,7 @@ template <> struct std::formatter<S> {
     char c = get_char();
     if (!isdigit(c) || (++iter, get_char()) != '}')
       throw format_error("invalid format");
-    width_arg_id = c - '0';
+    width_arg_id = static_cast<decltype(width_arg_id)>(c - '0');
```

Comment:
Please use `fmt::detail::to_unsigned` instead of a cast.

Reply:
Fixed